test/mocks/overrides/override_home_view_model.dart

```diff
@@ -8,7 +8,7 @@ import "../data/post_overview.dart";
 /// This class is particularly useful for the UI tests where we want to expose
 /// particular data to the views.
 /// By default it exposes an empty list of [PostOverview] and does nothing on refresh.
-class MockHomeViewModel extends AsyncNotifier<List<PostOverview>>
+class MockHomeViewModel extends AutoDisposeAsyncNotifier<List<PostOverview>>
```

Comment:
This appears to work when I tested it by hand by changing points in the database. I suggest we add a test that does this. We can either wait for #193 to be merged to use challenges, or instead change the points value in the fake database before navigating back to the profile.

Reply:
Yes, absolutely.

The test was added in 6a0d070 :)